README.md

```diff
@@ -43,6 +43,30 @@ function isDisposableEmail($email, $blocklist_path = null) {
     return in_array($domain, $disposable_domains);
 }
 ```
+
+**Go** contributed by [@pjebs](https://github.com/pjebs)
+
+```go
+import ("bufio"; "fmt"; "os"; "strings";)
+var disposableList = make(map[string]struct{}, 3500)
+func init() {
+	f, _ := os.Open("disposable_email_blocklist.conf")
+	for scanner := bufio.NewScanner(f); scanner.Scan(); {
+		disposableList[scanner.Text()] = struct{}{}
+	}
+	f.Close()
+}
+
+func isDisposableEmail(email string) (disposable bool) {
+	segs := strings.Split(email, "@")
+	_, disposable = disposableList[strings.ToLower(segs[len(segs)-1])]
+	return
+}
+```
+
+Alternatively, [anti-disposable-email](https://github.com/rocketlaunchr/anti-disposable-email) which auto-updates based on this repo.
```

Comment:
I would rather not endorse a project with a custom license (https://github.com/rocketlaunchr/anti-disposable-email/blob/master/LICENSE)

Reply:
It's practically the MIT license with an extra clause.

Reply:
It is practically NOT the MIT license.

Reply:
Okay I'll change it to MIT

Reply:
Done

Reply:
@martenson will this get accepted?